appcircle_jira/2.0.0/component.yaml

```diff
@@ -0,0 +1,65 @@
+platform: Common
+buildPlatform:
+displayName: Jira Comment
+description: "Add comment to Jira Issue and optionally change the status."
+docsUrl: https://docs.appcircle.io/integrations/jira-integration
+requiredComponents: "appcircle_git_clone"
+precedingComponents:
+followingComponents:
+webUrl: https://github.com/appcircleio/appcircle-jira-component
+repoUrl: https://github.com/appcircleio/appcircle-jira-component.git
+commit: d8c2e11
+inputs:
+- key: "AC_JIRA_HOST"
+  defaultValue: "$AC_JIRA_HOST"
+  isRequired: true
+  title: Jira Host
+  description: "Your Jira subdomain. Example: `mysubdomain.atlassian.net`"  
+  helpText:
+- key: "AC_JIRA_EMAIL"
+  defaultValue: "$AC_JIRA_EMAIL"
+  isRequired: false
+  title: Jira Email
+  description: "The email associated with your Jira account. This field is required for using API tokens instead of PAT."
+  helpText:
+- key: "AC_JIRA_TOKEN"
+  defaultValue: "$AC_JIRA_TOKEN"
+  isRequired: false
+  title: API Token
+  description: "User's API Token. If this value is fill, the Jira e-mail field must be filled. Only Jira Cloud users can use API Token. You can create token from [here](https://id.atlassian.com/manage-profile/security/api-tokens)."
+  helpText: "Please add this using **locked** environment variables."
+- key: "AC_JIRA_PAT"
+  defaultValue: "$AC_JIRA_PAT"
+  isRequired: false
+  title: Personal Access Token
+  description: "Specify the Personal Access Token for Jira authentication. Only Jira On-Prem users can use PAT."
+  helpText: "Please add this using **locked** environment variables."
+- key: "AC_JIRA_ISSUE"
+  defaultValue: "$AC_JIRA_ISSUE"
+  isRequired: true
+  title: Jira Issue
+  description: "The ID or key of the issue. Please check [documentation](https://docs.appcircle.io/integrations/jira-integration/) to learn how you can get this information from branch names or commit messages."
+  helpText:
+- key: "AC_JIRA_FAIL_TRANSITION"
+  defaultValue: "$AC_JIRA_FAIL_TRANSITION"
+  isRequired: false
+  title: Error Transition
+  description: "Transition ID or name for the failed step. Optionally change the status of your issue if the previous state fails. Ensure that the `Always run this step even if the previous steps fail` switch is turned for this feature to work."
+  helpText:
+- key: "AC_JIRA_SUCCESS_TRANSITION"
+  defaultValue: "$AC_JIRA_SUCCESS_TRANSITION"
+  isRequired: false
+  title: Success Transition
+  description: "Transition ID or name for the successful step. If the previous state succeeds, you can optionally change the status of your issue."
+  helpText:
+- key: "AC_JIRA_TEMPLATE_V2"
+  defaultValue: "Commit ID: $AC_GIT_COMMIT | Git Branch: $AC_GIT_BRANCH | Commit Message: $AC_COMMIT_MESSAGE | Appcircle Workflow Name: $AC_WORKFLOW_NAME | Date (UTC): AC_JIRA_DATE" 
```

Comment:
@ftmzhrsckldr I think, It may be text area instead of textFlield. Like version 3.0.0. I know it is not a Json but it would be better I think. 

@csonuryilmaz what do you think

Reply:
@boztopuz I agree, that (textarea) might be more comfortable for the end-user experience, but we can postpone it as a UI refactor.

Since we have some significant delays in delivery, it will be better for us to go on with the required updates up to now.